### ISSUE TITLE
Could NOT find Boost

### ISSUE THREAD

**Issue by kikunayar** [alembic/alembic#451]
```
      -- Could NOT find Boost (missing: Boost_INCLUDE_DIR python312) (Required is at least version "1.53.0")
      -- Could NOT find Boost (missing: Boost_INCLUDE_DIR python3.12) (Required is at least version "1.53.0")
      -- Could NOT find Boost (missing: Boost_INCLUDE_DIR) (Required is at least version "1.53.0")
```

**Comment by kikunayar** [comment]
I'm using python 3.12.1 is it the problem?

**Comment by kikunayar** [comment]
seem I installed the wrong boost

**Comment by kikunayar** [comment]
ahhh SET PATH

**Comment by lamiller0** [comment]
Yeah, you will need to make sure Cmake can find your boost install.  What Imath are you using?  You may run into problems based on this:  https://github.com/AcademySoftwareFoundation/Imath/issues/395

**Comment by kikunayar** [comment]
> Yeah, you will need to make sure Cmake can find your boost install. What Imath are you using? You may run into problems based on this: [AcademySoftwareFoundation/Imath#395](https://github.com/AcademySoftwareFoundation/Imath/issues/395)

Yes i using latest version
and i run into  the problems
so i change the version to 3.1.8 
and found the same problems
